Kitodo/src/main/resources/mapping.json

```
@@ -302,6 +302,9 @@
         "process": {
             "dynamic": "strict",
             "properties": {
+                "id": {
+                    "type": "text"
```

```suggestion
                    "type": "long"
```
`id` should be of type `long` because `text` does not work with default settings:

> Fielddata is disabled on text fields by default. Set fielddata=true on [id] in order to load fielddata in memory by uninverting the inverted index. Note that this can however use significant memory.